agent/agent-bootstrap/build.gradle.kts

```diff
@@ -8,24 +8,4 @@ dependencies {
   // TODO (heya) remove this when updating to upstream micrometer instrumentation
   compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api")
   compileOnly("io.opentelemetry:opentelemetry-semconv")
-  compileOnly("io.opentelemetry.instrumentation:opentelemetry-instrumentation-api-semconv")
-
-  compileOnly("com.google.auto.value:auto-value-annotations")
-  annotationProcessor("com.google.auto.value:auto-value")
-
-  implementation("ch.qos.logback:logback-classic")
-  implementation("ch.qos.logback.contrib:logback-json-classic")
-
-  // not using gson because it has dependency on java.sql.*, which is not available in Java 9+ bootstrap class loader
-  // only complaint so far about moshi is that it doesn"t give line numbers when there are json formatting errors
-  implementation("com.squareup.moshi:moshi")
```

Comment:
this was what motivated moving this code out of the bootstrap module (in addition to it being a generally good thing to minimize amount of classes in the bootstrap module)

(although this comment is now very outdated, and we'll move to jackson)